examples/data-objects/table-document/src/slice.ts

```diff
@@ -12,6 +12,9 @@ import { ConfigKey } from "./configKey.js";
 import { TableDocument } from "./document.js";
 import { ITable, TableDocumentItem } from "./table.js";
 
+/**
+ * @internal
+ */
```

Comment:
Keep this?

Reply:
Removed to minimize potential confusion.  Thx.